#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_drmpsb.dpatch by  <jbs@jbs-laptop>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: make drm_psb.ko depend of psb.ko

@DPATCH@

diff -p -up psb-kernel-source-4.41.1/Makefile.kernel.drmpsb psb-kernel-source-4.41.1/Makefile.kernel
--- psb-kernel-source-4.41.1/Makefile.kernel.drmpsb	2009-04-16 05:25:02.000000000 +0200
+++ psb-kernel-source-4.41.1/Makefile.kernel	2009-09-23 11:56:26.000000000 +0200
@@ -7,7 +7,7 @@
 # $XFree86: xc/programs/Xserver/hw/xfree86/os-support/linux/drm/kernel/Makefile.kernel,v 1.18 2003/08/16 17:59:17 dawes Exp $
 #
 
-drm-objs    := drm_auth.o drm_bufs.o drm_context.o drm_dma.o drm_drawable.o \
+drm-psb-objs    := drm_auth.o drm_bufs.o drm_context.o drm_dma.o drm_drawable.o \
 		drm_drv.o drm_fops.o drm_ioctl.o drm_irq.o \
 		drm_lock.o drm_memory.o drm_proc.o drm_stub.o drm_vm.o \
 		drm_sysfs.o drm_pci.o drm_agpsupport.o drm_scatter.o \
@@ -49,7 +49,7 @@ xgi-objs    := xgi_cmdlist.o xgi_drv.o x
 		xgi_fence.o
 
 ifeq ($(CONFIG_COMPAT),y)
-drm-objs    += drm_ioc32.o
+drm-psb-objs    += drm_ioc32.o
 radeon-objs += radeon_ioc32.o
 mga-objs    += mga_ioc32.o
 r128-objs   += r128_ioc32.o
@@ -58,7 +58,7 @@ nouveau-objs += nouveau_ioc32.o
 xgi-objs    += xgi_ioc32.o
 endif
 
-obj-m			+= drm.o
+obj-m			+= drm-psb.o
 obj-$(CONFIG_DRM_TDFX)	+= tdfx.o
 obj-$(CONFIG_DRM_R128)	+= r128.o
 obj-$(CONFIG_DRM_RADEON)+= radeon.o
